architecture/infrastructure_components/management_console.adoc

```diff
@@ -10,12 +10,29 @@
 toc::[]
 
 == Overview
-The OpenShift Management Console is a web-based front end for using OpenShift. It is started as part of the link:../architecture/kubernetes_infrastructure.html#master[master] and by default listens one port higher than the API server. All static assets required to run the console are served from the `openshift` binary.
+The OpenShift Management Console is a web-based front end for using OpenShift.
+Developers can link:../../dev_guide/console.html[use the Management
+Console] to visualize and browse the contents of their projects.
 
```

Comment:
Let's make the management console a user interface, rather than "web-based front end".